platform/darwin/src/MGLStyleLayer.h

```diff
@@ -18,6 +18,9 @@ NS_ASSUME_NONNULL_BEGIN
  subclasses of this class. Instead, create instances of
  `MGLBackgroundStyleLayer` and the concrete subclasses of
  `MGLForegroundStyleLayer`.
+ 
+ Do not add `MGLStyleLayer` objects to the style property of a `MGLMapView` before
```

Comment:
`style` should have code ticks around it, too, to make clear that we’re referring to a literal property.